underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -300,25 +298,14 @@ public OutputStream create(String path, CreateOptions options) throws IOExceptio
 
   @Override
   public OutputStream createDirect(String path, CreateOptions options) throws IOException {
-    IOException te = null;
     FileSystem hdfs = getFs();
-    RetryPolicy retryPolicy = new CountingRetry(MAX_TRY);
-    while (retryPolicy.attempt()) {
-      try {
-        // TODO(chaomin): support creating HDFS files with specified block size and replication.
-        OutputStream outputStream = new HdfsUnderFileOutputStream(
-            FileSystem.create(hdfs, new Path(path),
-              new FsPermission(options.getMode().toShort())));
-        if (options.getAcl() != null) {
-          setAclEntries(path, options.getAcl().getEntries());
-        }
-        return outputStream;
-      } catch (IOException e) {
-        LOG.warn("Attempt count {} : {} ", retryPolicy.getAttemptCount(), e.toString());
-        te = e;
-      }
+    // TODO(chaomin): support creating HDFS files with specified block size and replication.
```

Comment:
Not in the scope of this PR, but it's probably better to just remove this `TODO` since it's not likely to be done anyways.